### PR TITLE
[DCC] Move AssetImporters out of Experimental namespace for 2020.2

### PR DESCRIPTION
**Pull request by Thomasch-unity3d**
# **Please read;**
## **PR Workflow for the Graphics repository:**
* **All PRs must be opened as draft initially**
* Reviewers can be added while the PR is still in draft
* The PR can be marked as “Ready for Review” once the reviewers have confirmed that **no more changes are needed**
* Tests will start automatically after the PR is marked as “Ready for Review”
* **Do not use [skip ci]** - this can break some of our tooling
* Read the [Graphics repository & Yamato FAQ](http://go/graphics-yamato-faq).

### Checklist for PR maker
- [ ] Have you added a backport label (if needed)? For example, the `need-backport-*` label. After you backport the PR, the label changes to `backported-*`.
- [ ] Have you updated the changelog? Each package has a `CHANGELOG.md` file.
- [ ]  Have you updated or added the documentation for your PR? When you add a new feature, change a property name, or change the behavior of a feature, it's best practice to include related documentation changes in the same PR. If you do add documentation, make sure to add the relevant Graphics Docs team member as a reviewer of the PR. If you are not sure which person to add, see the [Docs team contacts sheet](https://docs.google.com/spreadsheets/d/1rgUWWgwLFEHIQ3Rz-LnK6PAKmbM49DZZ9al4hvnztOo/edit#gid=1058860420).
- [ ] Have you added a graphic test for your PR (if needed)? When you add a new feature, or discover a bug that tests don't cover, please add a graphic test.

---
### Purpose of this PR
Why is this PR needed, what hard problem is it solving/fixing?

---
### Testing status

**Manual Tests**: What did you do?
- [ ] Opened test project + Run graphic tests locally
- [ ] Built a player
- [ ] Checked new UI names with UX convention
- [ ] Tested UI multi-edition + Undo/Redo + Prefab overrides + Alignment in Preset
- [ ] C# and shader warnings (supress shader cache to see them)
- [ ] Checked new resources path for the reloader (in developer mode, you have a button at end of resources that check the paths)
- Other: 

**Automated Tests**: What did you setup? (Add a screenshot or the reference image of the test please)

**Yamato**: (Select your branch):
https://yamato.prd.cds.internal.unity3d.com/jobs/902-Graphics

Any test projects to go with this to help reviewers?

---
### Comments to reviewers
Notes for the reviewers you have assigned.

### CODE REVIEW

File: com.unity.render-pipelines.high-definition/Editor/AssetProcessors/FBXArnoldSurfaceMaterialDescriptionPreprocessor.cs [hunk 1]
```diff
@@ -1,7 +1,10 @@
 using System.IO;
 using UnityEditor.AssetImporters;
 using UnityEngine;
+#if !UNITY_2020_2_OR_NEWER
```
Comment (sebastienlagarde) [inline com.unity.render-pipelines.high-definition/Editor/AssetProcessors/FBXArnoldSurfaceMaterialDescriptionPreprocessor.cs:4]: should be

#if UNITY_2020_2_OR_NEWER
using UnityEditor.AssetImporters;
#else
using UnityEditor.Experimental.AssetImporters;	using UnityEditor.Experimental.AssetImporters;
#endif

Comment (sebastienlagarde) [reply com.unity.render-pipelines.high-definition/Editor/AssetProcessors/FBXArnoldSurfaceMaterialDescriptionPreprocessor.cs:4]: same for other stuff

File: com.unity.visualeffectgraph/Editor/Utilities/VectorFieldImporter/Editor/VectorFieldImporterEditor.cs [hunk 2]
```diff
@@ -1,10 +1,14 @@
 using UnityEngine;
-
+#if UNITY_2020_2_OR_NEWER
+using UnityEditor.AssetImporters;
+#else
```
Comment (sebastienlagarde) [inline com.unity.visualeffectgraph/Editor/Utilities/VectorFieldImporter/Editor/VectorFieldImporterEditor.cs:4]: thanks for fixing the other stuff!

File: com.unity.render-pipelines.high-definition/Editor/AssetProcessors/FBXArnoldSurfaceMaterialDescriptionPreprocessor.cs (hunk 1 above)
Comment (Thomasch-unity3d) [reply com.unity.render-pipelines.high-definition/Editor/AssetProcessors/FBXArnoldSurfaceMaterialDescriptionPreprocessor.cs:4]: There's already a 'using UnityEditor.AssetImporters;' above in this file, should be alright, no ? I'm getting my hands on a Unity build of the branch that does the migration to confirm..

Comment (sebastienlagarde) [reply com.unity.render-pipelines.high-definition/Editor/AssetProcessors/FBXArnoldSurfaceMaterialDescriptionPreprocessor.cs:4]: that's exactly my point, we should move the one above in the gaurd band, I will fix it :)

Comment (sebastienlagarde) [reply com.unity.render-pipelines.high-definition/Editor/AssetProcessors/FBXArnoldSurfaceMaterialDescriptionPreprocessor.cs]: done

**Comment by sebastienlagarde** [comment]
waiting for Yamato

**Pull request merged**